base/libgit2/callbacks.jl

```diff
@@ -29,76 +29,154 @@ end
 Function provides different credential acquisition functionality w.r.t. a connection protocol.
 If payload is provided then `payload_ptr` should contain `LibGit2.AbstractPayload` object.
 
-For `LibGit2.Consts.CREDTYPE_USERPASS_PLAINTEXT` type, if payload contains fields: `user` & `pass` they are used to create authentication credentials.
-In addition, if payload has field `used` it can be set to `true` to indicate that payload was used and abort callback with error. This behavior is required to avoid repeated authentication calls with incorrect credentials.
+For `LibGit2.Consts.CREDTYPE_USERPASS_PLAINTEXT` type, if payload contains fields:
+`user` & `pass` they are used to create authentication credentials.
+In addition, if payload has field `used` it can be set to `true` to indicate that payload was used
+and abort callback with error. This behavior is required to avoid repeated authentication calls with incorrect credentials.
```

Comment:
What do you mean by "to indicate that payload was used and abort callback with error"? If the field is set to true, where does the callback get aborted?